.github/workflows/cloud_java_client_test.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
         with:
           java-version: 8
           
-      - name: Run cloud tests
+      - name: Run cloud tests for UAT
```

Comment:
Will this work with the recent changes on the remote controller? I mean we made some changes for the viridian, so I am wondering if they apply to UAT as well?

Reply:
removed all uat and prod start rc steps